controllers/matchesController.js

```diff
@@ -69,130 +57,168 @@ module.exports = {
 
     Job.findById(jobId).then(job => {
       // sort the keywords by importance
-      let sortedHardSkills = getSortedKeywords("hard_skills.name", job.hard_skills);
-      let sortedSoftSkills = getSortedKeywords("soft_skills.name", job.soft_skills);
+      let sortedHardSkills = module.exports.getSortedKeywords("hard_skills.name", job.hard_skills);
```

Comment:
just wondeting, why we need module.exports here if this function is in the block module.exports = {...}? IT does not work like this, if we require matchesController  in another class and call on this?

Reply:
Yeah whenever I tried to call a function from module.exports of the same file, it always said it's not a function. That's why in the beginning of matchesController, we had all those helper functions _outside_ of module exports. But after some googling I found out that you had to call it like that -> `module.exports.XXX()` when it's from the same controller :) and outside of it you just import the controller in a variable and call it like `matchesController.XXX()`